lib/Optimizer/Partition.cpp

```diff
@@ -114,10 +115,10 @@ NodeFunctionMap selectBasicBlockPartitions(Function *F) {
 
 /// Given a function \p F and partitioning \p mapping, \return a FunctionDAG
 /// that contains appropriately-partitioned functions and their dependences.
-FunctionDAG doPartitioning(Function *F, NodeFunctionMap &mapping) {
+FunctionDAG doPartitioning(Function *F, NodeFunctionMap &mapping,
```

Comment:
Conceptually partitioning should be independent of a specific context. Can we initialize (add a backing tensor) outside of the partitioning method?

Reply:
Yep, done.  (I think I'd originally plumbed Context through only to perform `createSave`, but I've realized there's no good reason to do that.)